wiki/PlayersManual.md

```diff
@@ -29,41 +29,41 @@ The Star Barge is a slow ship, but with ample cargo capacity, enough to be able
 
 The Sparrow is the cheapest interstellar ship you can buy that is designed for combat. With enough luck or skill, a Sparrow pilot can earn income by disabling and boarding small pirate ships... or by turning pirate and attacking civilians. Once you earn a reputation in combat, you can find a more steady income as an escort for merchant caravans, but even so, life at the helm of a Sparrow is risky and uncertain. The safe way to make money in a Sparrow is by mining asteroids, which is described later.<br clear="all">
 
-Your game is saved automatically every time you take an action while landed. The game also maintains the three most recent previous saves in case you need to revert to one of them (via the &ldquo;Load / Save...&rdquo; menu). If you quit the game while landed on a planet, your progress is saved; otherwise, your progress since the last planet you departed from is lost. If you want to be able to return to a specific point in time, perhaps because you are about to make a major decision or undertake a risky mission, you can save a &ldquo;snapshot&rdquo; of the current game. These snapshots are never overwritten or deleted unless you choose to delete them. A special snapshot named &ldquo;autosave&rdquo; is updated whenever you reach a new milestone in the story line.
+Your game is saved automatically every time you take an action while landed. The game also maintains the three most recent previous saves in case you need to revert to one of them (via the "Load / Save" menu). If you quit the game while landed on a planet, your progress is saved; otherwise, your progress since the last planet you departed from is lost. If you want to be able to return to a specific point in time, perhaps because you are about to make a major decision or undertake a risky mission, you can save a "snapshot" of the current game. These snapshots are never overwritten or deleted unless you choose to delete them. A special snapshot named "autosave" is updated whenever you reach a new milestone in the story line.
```

Comment:
```suggestion
Your game is saved automatically every time you take an action while landed. The game also maintains the three most recent previous saves in case you need to revert to one of them (via the "Load / Save..." menu). If you quit the game while landed on a planet, your progress is saved; otherwise, your progress since the last planet you departed from is lost. If you want to be able to return to a specific point in time, perhaps because you are about to make a major decision or undertake a risky mission, you can save a "snapshot" of the current game. These snapshots are never overwritten or deleted unless you choose to delete them. A special snapshot named "autosave" is updated whenever you reach a new milestone in the story line.
```
The ellipses are present on the button text.

Reply:
I am aware, but it hadn't been there before, so I didn't want to change it without approval.